tools/eslint-config-web3-base/rules/check-header.js

```diff
@@ -0,0 +1,57 @@
+const fs = require('fs');
+const path = require('path');
+
+module.exports = {
+	meta: {
+		fixable: 'code', // Add rule as fixable (--fix at cli)
+		files: ['*.ts', '*.tsx'],
+	},
+	create(context) {
+		const licenseRaw = readLicenseFile(path.join(__dirname, '../../../LICENSE')).trim();
+		const license = removeWhitespaces(licenseRaw).trim();
+		const licenseInComments = `\/*\n${licenseRaw}\n*\/\n\n`; // Wrap license in multiline comments - For fix-
+
+		const hasLicense = comment => comment.indexOf(license) > -1; // Is license present in comment?
+
+		return {
+			Program(node) {
+				const comments = context.getAllComments().map(cleanComment);
+				const sourceCode = context.getSourceCode();
+				if (!comments.some(hasLicense)) {
```

Comment:
This rule will pass even if license is put anywhere in the middle or bottom of the file. Where as we want the license to be exactly on the top.